airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/record_buffer/FileBuffer.java

```diff
@@ -47,7 +47,7 @@ public FileBuffer(final String fileExtension) {
   @Override
   public OutputStream getOutputStream() throws IOException {
     if (outputStream == null || tempFile == null) {
-      tempFile = Files.createTempFile(UUID.randomUUID().toString(), fileExtension).toFile();
+      tempFile = Files.createTempFile("", fileExtension).toFile();
```

Comment:
nit: after this change java.util.UUID import becomes unused, so it should be removed